src/task2.js

```diff
@@ -0,0 +1,45 @@
+const csv = require('csvtojson');
+const fs = require('fs');
+
+const writeText = fs.createWriteStream('node_mentoring_t1_2_input_example.txt');
+const results = [];
+
+var s = fs.createReadStream('node_mentoring_t1_2_input_example.csv')
+        .pipe(csv())
```

Comment:
Consider using a pipeline standard utility for this task.
https://nodejs.org/api/stream.html#stream_stream_pipeline_streams_callback
That way you can better organise your code in a more functional fashion.